.github/workflows/ci.yaml

```diff
@@ -344,10 +344,26 @@ jobs:
         if: steps.npm-e2e-cache.outputs.cache-hit != 'true'
         run: cd test/e2e && npm install
 
+  prepare:
+    runs-on: ubuntu-20.04
+    outputs:
+      matrix: ${{ steps.prepare.outputs.matrix }}
+    steps:
+      - name: Create matrix ⊹
+        id: prepare
+        uses: bahmutov/gh-build-matrix@main
+        with:
+          n: 4
```

Comment:
Can we have this `4` in some variable on top?
So we can change if needed, as it is used in 2 places

Reply:
Also: where does this number come from? Should it be the number of cores?